doc/btp_cap.txt

```diff
@@ -0,0 +1,269 @@
+CAP Service (ID 24)
+===================
+
+Commands and responses:
+
+	Opcode 0x00 - Error response
+
+	Opcode 0x01 - Read Supported Commands command/response
+
+		Controller Index:	<non-controller>
+		Command parameters:	<none>
+		Response parameters:	<supported commands> (variable)
+
+		Each bit in response is a flag indicating if command with
+		opcode matching bit number is supported. Bit set to 1 means
+		that command is supported. Bit 0 is reserved and shall always
+		be set to 0. If specific bit is not present in response (less
+		than required bytes received) it shall be assumed that command
+		is not supported.
+
+		In case of an error, the error response will be returned.
+
+	Opcode 0x02 - Discover and Subscribe
+		Controller Index:	<controller id>
+		Command parameters:	Address_Type (1 octet)
+					Address (6 octets)
+		Response parameters:	<none>
+
+		This command is used to discover all remote PACS, ASCS and BASS
+		characteristics. In case of an error, the error status response
+		will be returned. In case of a success, the IUT continues processing
+		the command asynchronously. During discovery, the IUT may send events:
+					Discover and Subscribe Completed event
+					Codec Capabilities Found event
+					ASE Found event
+
+	Opcode 0x03 - Ordered Access
+		Controller Index:	<controller id>
+		Command parameters:	Address_Type (1 octet)
+					Address (6 octets)
+		Response parameters:	<none>
+
+		This command is used to execute the CSIP Ordered Access.
+		In case of an error, the error status response will be returned.
+		In case of a success, the IUT continues processing
+		the command asynchronously.
+
+	Opcode 0x04 - Unicast Setup Stream
+		Controller Index:	<controller id>
+		Command parameters:	Address_Type (1 octet)
+					Address (6 octets)
+					ASE_ID (1 octet)
+					CIS_ID (1 octet)
+					CIG_ID (1 octet)
+					Coding Format (1 octet)
+					VID (2 octets)
+					CID (2 octets)
+					Sdu Interval (3 octets)
+					Framing (1 octet)
+					Max SDU (2 octets)
+					Retransmission Num (1 octet)
+					Max Transport Latency (2 octets)
+					Presentation Delay (3 octets)
+					Codec Config LTVs length (1 octet)
+					Metadata LTVs length (1 octet)
+					LTVs (varies)
+
+		Response parameters:	<none>
+
+		This command is used to configure CISes locally before executing
```

Comment:
As this command takes single CIS as parameter, I would say it's `used to configure CIS` and mention that it can be called multiple times per ASE.
This raises a question whether this shouldn't be called `Unicast Setup ASE` as it can be called once per ASE (while twice per CIS, as as the Sink/Source ASE might share single CIS)